pensando_psm/info.json

```diff
@@ -0,0 +1,544 @@
+                "visible": true,
+                "editable": true,
+                "value": "default"
+            },
+            {
+                "title": "Username",
+                "type": "text",
+                "name": "username",
+                "required": true,
+                "visible": true,
+                "editable": true
+            },
+            {
+                "title": "Password",
+                "type": "password",
+                "name": "password",
+                "required": true,
+                "visible": true,
+                "editable": true
+            },
+            {
+                "title": "Protocol",
+                "type": "select",
+                "name": "protocol",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "value": "HTTPS",
+                "options": [
+                    "HTTP",
+                    "HTTPS"
+                ]
+            },
+            {
+                "title": "Verify SSL",
+                "type": "checkbox",
+                "name": "verify_ssl",
+                "required": false,
+                "visible": true,
+                "editable": true
+            }
+        ]
+    },
+    "operations": [
+        {
+            "operation": "debug_remove_session_state",
+            "title": "debug - remove session state",
+            "description": "Debug login by removing session state from disk.",
+            "enabled": false,
+            "category": "",
+            "annotation": "",
+            "output_schema": {
+                "result": "",
+                "api_data": ""
+            }
+        },
+        {
+            "operation": "debug_reset_session_state",
+            "title": "debug - reset session state",
+            "description": "Debug login by resetting session state on disk.",
+            "enabled": false,
+            "category": "",
+            "annotation": "",
+            "output_schema": {
+                "result": "",
+                "api_data": ""
+            }
+        },
+        {
+            "operation": "debug_expire_cookie",
+            "title": "debug - expire auth cookie",
+            "description": "Debug login by expiring the auth cookie on disk.",
+            "enabled": false,
+            "category": "",
+            "annotation": "",
+            "output_schema": {
+                "result": "",
+                "api_data": ""
+            }
+        },
+        {
+            "operation": "get_network_security_policies",
+            "title": "Get Network Security Policies",
+            "description": "Get the Network Security Policy list.",
+            "enabled": true,
+            "category": "",
+            "annotation": "",
+            "output_schema": {
+                "result": "",
+                "api_data": ""
+            }
+        },
+        {
+            "operation": "get_alerts",
```

Comment:
Please add pagination parameters and filter parameters if available.

Reply:
I'm not sure how to do this. Could you point me to some documentation? Thanks!